app/helpers/schedule_helper.rb

```diff
@@ -8,7 +8,7 @@ def row_time(time_slot)
   end
 
   def grid_data
-    tracks = current_event.tracks.sort_by_name.pluck(:name).map(&:parameterize)
+    tracks = current_event.tracks.sort_by_name.pluck(:name).map{|name| truncate(name, length: 30).parameterize}
```

Comment:
Do we know if this needs to be truncated?  It appears it's used for css track classes?  If so this change might break styles.